### ISSUE TITLE
E2E Login For Updated Landing Page

### ISSUE THREAD

**Issue by Elaine-Krauss-TCG** [fecgov/fecfile-web-app#1693]
Business Reason

When the nightly End-to-End tests run, there must be a form allowing them to log in via the legacy login method.

Acceptance Criteria

***Given*** that you are on an environment where username/password login is enabled

  ***When*** you visit the login page

  ***Then*** you will see a Debug Login form

*   [x] The debug login form will only appear if the API allows legacy logins

### QA Notes ###
- [ ] BA/Design Quality Checklist complete

- There should be Committee ID, Email, and Password fields on the login page if the environment you're in is configured to show it.

### DEV Notes ###
- [x] The debug login form should be isolated in its own component
- [x] The API should only allow legacy logins if an environment variable is set (implemented in a prior ticket)

### Design ###
null

[FECFILE-493](https://fecgov.atlassian.net/browse/FECFILE-493)

**Comment by Elaine-Krauss-TCG** [comment]
Todd graciously handled the API changes needed for this ticket in [this ticket](https://github.com/fecgov/fecfile-web-api/pull/691)

**Comment by mjtravers** [comment]
This ticket is dependent on #129. Placing it on hold until #129 is back in CR.

**Comment by mjtravers** [comment]
Passes CR. Sending to QA.

**Comment by mjtravers** [comment]
QA Notes: At the moment, the e2e login is hidden on DEV. After you verify this, we can turn on the e2e login for that to be verified as well.

**Comment by WiseQA** [comment]
QA verified the "Debug Login Form" is displayed for e2e testing.   Per DEV this will be hidden from the landing page and ONLY used for e2e testing.

<img width="838" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOEFkQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--54bc2a7c97cfaec528dec7972bf3867b00322901/image.png" alt="image.png" />

QA Review Completed.   Moved to Stage Ready.
